.github/workflows-source/ci-main.yml

```diff
@@ -249,12 +249,17 @@ jobs:
           nsc bazel cache setup --bazelrc=/tmp/bazel-cache.bazelrc
       - uses: actions/checkout@v4
 
-      - name: build
+      - name: Build and Test
```

Comment:
The name of the job (currently "Bazel Build arm64-linux") as well as comment could also be updated.